src/main/java/frc/robot/MotorController.java

```diff
@@ -20,21 +22,35 @@
     private String mName;
     private CANSparkMax mSparkMax;
     private CANEncoder mEncoder;
+    private CANPIDController mPIDController = null;
+    private double mP;
+    private double mI;
+    private double mD;
 
-    public MotorController(String name, int deviceId) {
+    public MotorController(String name, int deviceID) {
         mName = name;
-        mSparkMax = new CANSparkMax(deviceId, MotorType.kBrushless);
+        mSparkMax = new CANSparkMax(deviceID, MotorType.kBrushless);
         //Initializing encoder
         //We can only call .getEncoder once because of a bug in spark max api
         //which causes encoder initialization to occur on every call of .getEncoder() this breaks things
         mEncoder = mSparkMax.getEncoder();
         mSparkMax.restoreFactoryDefaults();
     }
 
-    public MotorController(String name, int deviceID, int smartCurrentLimit) {
-        this(name, deviceID); //calls the constructor above
+    //Todo: we must add constants for PID values once we have found the right values
+    public MotorController(String name, int deviceID, int smartCurrentLimit, boolean... enablePid) {
```

Comment:
The enablePID parameter is optional, so  we can call MotorController(name, deviceId, smartCurrent) already with out adding a constructor. I'm not sure what changing it would add.

Reply:
We cleaned up the constructors in the path branch to resolve another todo.